static/_app/tools/idle-idle-gamedev/css/style.css

```
@@ -0,0 +1,206 @@
+.task-content{
+    flex: 1 0 auto;
+}
+.hidden{
+    display: none!important;
+}
+.margin-auto{
+    margin-right: auto;
+    margin-left: auto;
+}
+.luts-header{
+    background: linear-gradient(0,#303030,#424242);
```

Biome (2.1.2)

[error] Unexpected nonstandard direction

You should fix the direction value to follow the syntax.
See MDN web docs for more details.

(lint/correctness/noInvalidDirectionInLinearGradient)